sklearn/datasets/openml.py

```diff
@@ -263,6 +265,58 @@ def _convert_arff_data(arff_data, col_slice_x, col_slice_y, shape=None):
         raise ValueError('Unexpected Data Type obtained from arff.')
 
 
+def _feature_to_dtype(feature):
+    """Map feature to dtype for pandas DataFrame
+    """
+    if feature["data_type"] == "string":
+        return object
+    elif feature["data_type"] == "nominal":
+        return 'category'
```

Comment:
we need to do more than this: the categorical ordering should be the same as defined in the ARFF, ideally.

Reply:
Having the order from the ARFF file makes sense. I’ll update this accordingly.